slides/src/content/a11y.md

```diff
@@ -0,0 +1,142 @@
+<!-- .slide: data-background="../content/images/title-slide.jpg" -->
+<!-- .slide: id="a11y" -->
+## Accessibility and Angular
+
+---
+<!-- .slide: id="a11y" -->
+## What is Accessibility
+
+Accessibility is generally defined as giving people of all abilities opportunities to participate fully in everyday life.  This more recently includes use of digital technology.
+
+Accessibility can often be thought of as being complimentary to, or a subset of usability.  A feature on a site may be highly usable but inaccessible.  
+
+- For example, a button may be easy to find and click, but inaccessible via a keyboard
+- This prevents visually impaired users from browsing a site that otherwise has high usability
+
+---
+<!-- .slide: id="a11y" -->
+## Accessibility and the Law
```

Comment:
1. Combine this with the slide following
2. Cut the detail significantly - developers won't care much about specifics.